vignettes/spellchecks.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "Spellchecks"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Spellchecks}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Spelling Workflow
+
+This GitHub Actions workflow performs spellchecking on the specified R files. The workflow is triggered manually (`workflow_dispatch`) and can also be invoked as a workflow call (`workflow_call`).
+
+## Workflow Structure
+
+The workflow consists of a single job: `roxygen`.
```

Comment:
Why is the job called `roxygen`?

Reply:
(indeed this is an error .. I'll update this in my next commit)